intellij-ext/modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordApi.kt

```diff
@@ -49,10 +49,14 @@ object MirrordApi {
     fun listPods(configFile: String?, project: Project?, wslDistribution: WSLDistribution?): List<String> {
         MirrordLogger.logger.debug("listing pods")
         var commandLine = GeneralCommandLine(cliPath(wslDistribution), "ls", "-o", "json")
-        configFile?.let {
+        configFile?.let {config ->
             MirrordLogger.logger.debug("adding configFile to command line")
             commandLine.addParameter("-f")
-            commandLine.addParameter(it)
+            wslDistribution?.let {
+                commandLine.addParameter(it.getWslPath(config) ?: config)
```

Comment:
Are both options valid?

Reply:
no but I think it's a worthy fallback - my concern that path is already `WSL'd` for some reason and that's why it didn't work, hence fallback

Reply:
added clarification